src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html

```diff
@@ -0,0 +1,7 @@
+<ScrollView orientation="horizontal">
+    <WrapLayout orientation="horizontal">
+        <StackLayout *ngFor="let item of items" (tap)="segmentedChipSelect(item)">
```

Comment:
Remove redundant wrapper:
```suggestion
        <kirby-chip *ngFor="let item of items" (tap)="segmentedChipSelect(item)" ...>
```

Reply:
Redudant wrapper removed

Reply:
We do not need this as it won't help as per Slack conversation.